hooks/pre-exit

```diff
@@ -23,7 +23,7 @@ fi
 ### Step 2: Run `Pkg.gc` to try to reduce the size of the depot.
 
 echo "Running Pkg.gc()"
-julia --color=yes -e 'using Pkg; using Dates; Pkg.gc(; collect_delay=Day(7), verbose=true)'
+julia --color=yes -e 'using Pkg; using Dates; Pkg.gc(; collect_delay=Day(30), verbose=true)'
```

Comment:
```
julia> Pkg.gc(; collect_delay=Month(1))
      Active manifest files: 6 found
ERROR: MethodError: no method matching isless(::Month, ::Millisecond)
```

:disappointed:

Reply:
Yeah. `Month` is not a fixed period, right? Some months have 30 days, some months have 31 days. So `Month` does not have a fixed definition in terms of milliseconds.

But e.g. `Day(30)` can be exactly defined in terms of a fixed definition of milliseconds (if you ignore leap seconds).

Reply:
Oh sure, it just doesn't make for a good user experience (because `Month <: Period`); naively I had expected it to work, but it makes sense.